tests/integration/it_owner_privilages_test.py

```diff
@@ -41,3 +39,102 @@ def test_can_access_own_employees(app):
                       registration_date=datetime.utcnow(), user_status="T",
                       email="test@test.com", password="bla")
     assert has_privilege(method=Method.READ, resource="employee")
+
+
+def test_can_manage_locations_from_same_company(app, db_session):
+    my_company = Company(
+        name="Acme Inc.", code="code1", address="addr"
+    )
+    db_session.add(my_company)
+    db_session.commit()
+    me = Employee(
+        first_name="Alice", last_name="Cooper",
+        username="alice", phone_number="1",
+        birth_date=datetime.utcnow(),
+        pin_code=7777,
+        account_status="on",
+        user_status="on",
+        registration_date=datetime.utcnow(),
+        company_id=my_company.id,
+        email="test@test.com", password="bla"
+    )
+    db_session.add(me)
+    flask.g.user = me
+    location = Location(
+        name="name",
+        code="123",
+        company_id=my_company.id,
+        country="US",
+        region="region",
+        city="city",
+        address="address",
+        longitude="123",
+        latitude="123",
+        type="type",
+        status="status"
+    )
+    db_session.add(location)
+    db_session.commit()
```

Comment:
@krzyk let's add instances to session and then make one `db_session.commit()`, instead of multiple commits

Reply:
@krzyk and the same in the second test

Reply:
@vryazanov not quite, I get id's only after I do commit, so e.g. if I want to use my_company.id, I need to commit it first

Reply:
@krzyk  Actually to get instance `id` need to execute `db_session.flush()`, it will interact with database, but transaction will not be commited. but anyway, I think we can leave it as it is